inspiration/resources-that-influence.qmd

```diff
@@ -88,48 +86,44 @@ https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008770
 
 > I've come to use the term "the architecture of participation" to describe the nature of systems that are designed for user contribution.
 
-## Systems Change: Donella Meadows 
+## Systems Change: Donella Meadows
 
 [Leverage points: places to intervene in a system](http://donellameadows.org/archives/leverage-points-places-to-intervene-in-a-system/): (in increasing order of effectiveness)
 
 12. Constants, parameters, numbers (such as subsidies, taxes, standards).
-11. The sizes of buffers and other stabilizing stocks, relative to their flows.
-10. The structure of material stocks and flows (such as transport networks, population age structures).
-9. The lengths of delays, relative to the rate of system change.
-8. The strength of negative feedback loops, relative to the impacts they are trying to correct against.
-7. The gain around driving positive feedback loops.
-6. The structure of information flows (who does and does not have access to information).
-5. The rules of the system (such as incentives, punishments, constraints).
-4. The power to add, change, evolve, or self-organize system structure.
-3. The goals of the system.
-2. The mindset or paradigm out of which the system — its goals, structure, rules, delays, parameters — arises.
-1. The power to transcend paradigms.
-
+13. The sizes of buffers and other stabilizing stocks, relative to their flows.
```

Comment:
I think this edit is not correct; the numbered list goes from 12 down to 1 not 12 to 23